packages/console/src/components/DetailsSkeleton/index.module.scss

```diff
@@ -46,42 +50,73 @@
     }
   }
 
+  .tabBar {
+    width: 100%;
+    height: 28px;
+    @include _.shimmering-animation;
+  }
+
   .content {
     flex: 1;
     display: flex;
-    flex-direction: column;
     width: 100%;
-    padding: _.unit(6);
-    margin-top: _.unit(6);
     border-radius: 16px;
     background-color: var(--color-layer-1);
+    padding: _.unit(6);
 
-    .tabBar {
-      @include _.shimmering-animation;
-      width: 100%;
-      height: 25px;
-      margin-bottom: _.unit(13.5);
-    }
+    .introduction {
+      width: 296px;
```

Comment:
just curious about those magic numbers, should they be aligned with component width or just a random 296 from figma?

Reply:
This follows the design settings.
<img width="223" alt="image" src="https://user-images.githubusercontent.com/10806653/207485737-86714699-4e47-48dd-a13d-1b5f2ff26f09.png">